resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/features/BasicDriverFeatureStep.scala

```diff
@@ -129,6 +129,12 @@ private[spark] class BasicDriverFeatureStep(conf: KubernetesDriverConf)
           .withNewFieldRef("v1", "status.podIP")
           .build())
         .endEnv()
+      .addNewEnv()
+        .withName(ENV_DRIVER_POD_NAME)
+        .withValueFrom(new EnvVarSourceBuilder()
+          .withNewFieldRef("v1", "metadata.name")
+          .build())
+        .endEnv()
```

Comment:
If we need this, this should be handled separately before this PR.

Reply:
removed